File: Homeworks/homework010.py
def writeFile():
    file = open("personal.dat", "w")
    
    while(True):
        emid = str(input("Employee Id: "))
        if(emid == "0"):
            break
        name = str(input("Employee First name and Last name: "))        
        age = str(input("Employee Age: "))
        salary = str(input("Employee Salary: "))
        department = str(input("Employee Department: "))

        file.write("%s,%s,%s,%s,%s\n" %(emid, name, age, salary, department))
        
        print("-----------------------------------------------------------")
    
    file.close()
    
    
def readFile():
    file = open("personal.dat", "r")
    print("-----------------------------------------------------------")
    while True:
        string = file.readline()
        if string == '':
            break
        print(string)
        
    file.close()

writeFile()
readFile()

File: Homeworks/homework010.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main(){
	ofstream file("personal.dat");
	
	while(true){
		string id, name, age, salary, department;
		
		cout << "Employee id: "; getline(cin, id);
		if(id == "0") break;
		cout << "Employee First name and Last name: "; getline(cin, name);
		cout << "Employee age: "; getline(cin, age);
		cout << "Employee salary: "; getline(cin, salary);
		cout << "Department: "; getline(cin, department);
		
		file << id + "," + name + "," + age + "," + salary + "," + department << endl;	
		
		cout << "---------------------------------------------------" << endl;
	}
	
	file.close();
	
	string mytext;
	
	ifstream readFile;
	readFile.open("personal.dat", ios::in);
	while(getline (readFile, mytext)){
		cout << mytext << endl;
	}
	
	file.close();
}
